src/main/webapp/manage-groups.xhtml

```diff
@@ -171,7 +171,7 @@
                             <p:commandButton id="saveGroup" styleClass="btn btn-default"
                                            actionListener="#{manageGroupsPage.saveExplicitGroup}"
                                            value="#{bundle.saveChanges}"
-                                           update="@all">
```

Comment:
I think this will also need the messagePanel.

Reply:
Fixed in 79301e6.